pyproject.toml

```diff
@@ -0,0 +1,59 @@
+[project]
+name = "bson"
+version = "0.5.10"
```

Comment:
I think we should keep this as is for now -- but the next step would be to introduce some dynamic versioning.

Hatch seems to prefer using its "version" command:

https://waylonwalker.com/hatch-version/

https://hatch.pypa.io/1.2/version/

Personally, I like there to be a `__version__` string in the `__init__.py`.

I know that's a bit controversial, but it doesn't really harm anything, as long as it's all dynamically managed -- I *think* you can do that with hatch version by:

```
[tool.hatch.version]
path = "bson/__init__.py"
```

but I haven't tested it.

Anyway, I don't think we need to hold up this PR for that -- it can come later with some more thought and experimentation.

Reply:
NOTE: just tested it, telling hatch to find the version in __init__.py works fine -- so I think this is the way to go.